Samples/PublicSamples/RecordingBot/docs/setup/policy.md

```diff
@@ -5,109 +5,121 @@ The following steps should be done by an Office 365 administrator.
-    $identity = Get-CsOnlineApplicationInstance | Where-Object {$_.DisplayName -Match "<displayName>"} | Select ObjectId
-    Sync-CsOnlineApplicationInstance `
-        -ObjectId $identity."ObjectId"
-    ```
-
-After initial set up you only need to run steps 1-3 in future sessions.
-
-## Create a Recording Policy
-
-Requires the application instance Object ID created above. Continue your powershell session and run the following commands.
-
-1. Create a new Teams recording policy for governing automatic policy-based recording in your tenant:
-
-    ```powershell
-    $policyName = '<policyIdentityName>'
-    New-CsTeamsComplianceRecordingPolicy `
-	    -Identity $policyName `
-	    -Enabled $true `
-	    -ComplianceRecordingApplications @(New-CsTeamsComplianceRecordingApplication -Parent $policyName -Id $identity."ObjectId")
-    ```
-
-    >Note: when you need to re-run this command for the same policy and/or the same identity, use `Set-` commands instead of `New-`, as in the following example:
-
-    ```powershell
-    $tag = 'Tag:' + $policyName + '/' + $identity."ObjectId"
-    Set-CsTeamsComplianceRecordingPolicy `
-        -Identity $policyName `
-        -Enabled $true
-    ```
-
-    >Note: if you ever need to remove the existing Teams Compliance Recording policy, you can execute the following command:
-
-    ```powershell
-    $tag = 'Tag:' + $policyName
-    Get-CsTeamsComplianceRecordingPolicy -Filter $tag | Remove-CsTeamsComplianceRecordingPolicy
-    ```
-
-After 30-60 seconds, the policy should show up. To verify your policy was created correctly:
-
-* `Get-CsTeamsComplianceRecordingPolicy <policyIdentityName>`
-
-    ```powershell
-    Get-CsTeamsComplianceRecordingPolicy $policyName
-    ```
-If the policy has been successfully created, you should see the output looks like this:
-![Policy Output](../images/policy_output.png)
-
-## Assign the Recording Policy
-
-Requries the policy identity created above. Contine your powershell session and run the following commands.
-
-1. Grant-CsTeamsComplianceRecordingPolicy
-
-    ```powershell
-    Grant-CsTeamsComplianceRecordingPolicy `
-        -Identity <upn@contoso.com> `
-        -PolicyName $policyName
-    ```
-
-    After a couple of minutes, to verify your policy was assigned correctly:
-
-2. Get-CsOnlineUser
-
-    ```powershell
-    Get-CsOnlineUser <upn@contoso.com> | ft sipaddress, tenantid, TeamsComplianceRecordingPolicy
-    ```
-    If your policy has been assigned correctly, you should see a similar output:
-
-    ![Policy Result](../images/policy-result.png)
+## Prerequisites
+
+- PowerShell (5.1, comes with Windows) as Administrator
+- PowerShell execution policy of at least `RemoteSigned`
+- PowerShell Module
+ - [SkypeForBusiness](https://learn.microsoft.com/en-us/powershell/module/skype/?view=skype-ps)
+ - or [MicrosoftTeams](https://learn.microsoft.com/en-us/powershell/module/teams/?view=teams-ps)
+- An Office 365 administrator
+- An Office 365 user account
```

Comment:
Why is both mentioned here admin and user but in the further documentation no further differentiation is made

Reply:
Removed user and noted that adding users to policies or vice versa is outside of this documentations scope